SB.js

```diff
@@ -99,18 +99,23 @@ function configProxy() {
     });
 	
     var handler = {
-        set: function(obj, prop, value) {
+        set(obj, prop, value) {
             SB.localConfig[prop] = value;
 
             chrome.storage.sync.set({
                 [prop]: encodeStoredItem(value)
             });
         },
-        get: function(obj, prop) {
+
+        get(obj, prop) {
             let data = SB.localConfig[prop];
             if(data instanceof Map) data = new MapIO(prop);
 
-			return obj[prop] || data;
+            return obj[prop] || data;
+        },
+	
+        deleteProperty(obj, prop) {
+	    chrome.storage.sync.remove(key);
```

Comment:
This code was using `key` instead of `prop` and still working since `key` wasn't scoped properly. `key` was a global variable created by the for loop. I fixed it though.